tap_xero/streams.py

```diff
@@ -178,39 +178,39 @@ def sync(self, ctx):
     PaginatedStream("credit_notes", ["CreditNoteID"], format_fn=transform.format_credit_notes),
     PaginatedStream("invoices", ["InvoiceID"]),
     PaginatedStream("manual_journals", ["ManualJournalID"]),
-    PaginatedStream("overpayments", ["OverpaymentID"], format_fn=transform.format_over_pre_payments),
-    PaginatedStream("prepayments", ["PrepaymentID"], format_fn=transform.format_over_pre_payments),
-    PaginatedStream("purchase_orders", ["PurchaseOrderID"]),
+    # PaginatedStream("overpayments", ["OverpaymentID"], format_fn=transform.format_over_pre_payments),
+    # PaginatedStream("prepayments", ["PrepaymentID"], format_fn=transform.format_over_pre_payments),
+    # PaginatedStream("purchase_orders", ["PurchaseOrderID"]),
 
     # JOURNALS STREAM
     # This endpoint is paginated, but in its own special snowflake way.
-    Journals("journals", ["JournalID"], bookmark_key="JournalNumber"),
+    # Journals("journals", ["JournalID"], bookmark_key="JournalNumber"),
 
     # NON-PAGINATED STREAMS
     # These endpoints do not support pagination, but do support the Modified At
     # header.
-    BookmarkedStream("accounts", ["AccountID"]),
-    BookmarkedStream("bank_transfers", ["BankTransferID"], bookmark_key="CreatedDateUTC"),
-    BookmarkedStream("employees", ["EmployeeID"]),
-    BookmarkedStream("expense_claims", ["ExpenseClaimID"]),
-    BookmarkedStream("items", ["ItemID"]),
+    Everything("accounts", ["AccountID"]),
```

Comment:
Does this work?

Reply:
yes

Reply:
reversed to `BookmarkerStream` since the issue was with `start_date` used in the config file